pkg/web/http_receiver_v2_test.go

```diff
@@ -45,7 +45,7 @@ func TestForwardingEndToEndV2(t *testing.T) {
 			ctx = clock.Context(ctx, mockClock)
 
 			ch := &channeledHandler{
-				chMaps: make(chan *gostatsd.MetricMap),
+				chMaps: make(chan *gostatsd.MetricMap, 1),
```

Comment:
We need this because the `sendNop` is now sync.  If that doesn't go in to the channel, `HttpForwarderHandlerV2.Run` never starts using the virtual clock, and `fixtures.NextStep()` waits forever (because it's waiting for something to use the clock)